common/src/main/java/com/microsoft/identity/common/internal/ui/webview/certbasedauth/DialogHolder.java

```diff
@@ -192,12 +209,25 @@ public synchronized boolean isDialogShowing() {
     }
 
     /**
-     * Runs the onCancelCbaCallback code for the current dialog.
-     * Used when YubiKey is unexpectedly disconnected from device.
+     * Informs if a {@link SmartcardRemovalPromptDialog} is currently showing.
+     *
+     * @return True if the current dialog showing is an instance of SmartcardRemovalPromptDialog. False otherwise.
+     */
+    @Override
+    public synchronized boolean isSmartcardRemovalPromptDialogShowing() {
+        if (mCurrentDialog != null) {
+            return mCurrentDialog instanceof SmartcardRemovalPromptDialog;
+        }
+        return false;
```

Comment:
Instance Of checks will return false if null
```suggestion
        return mCurrentDialog instanceof SmartcardRemovalPromptDialog;
```